ban/http/resources.py

```diff
@@ -16,38 +16,116 @@ class BaseCollection:
     DEFAULT_LIMIT = 20
     MAX_LIMIT = 100
 
-    def get_limit(self, req):
-        return min(int(req.params.get('limit', self.DEFAULT_LIMIT)),
-                   self.MAX_LIMIT)
+    def get_start(self, req, count):
+        """Get start value from request.
```

Comment:
Let's talk a bit about code documentation :)
Code documentation is good. Descriptive code documentation is bad (at best: useless).
Code should be self documenting: good naming, readable, split over named functions, not too much indented…
So code documentation should only add the missing bit: where the code can't say.
In other words, this is the mantra for code documentation: "Do not say what, say why" :)
You may not be OK with this in a first read, so let's discuss this more IRL asap :)